### ISSUE TITLE
New Schools Today: Update alt text for New Schools Today image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#4046]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `New Schools Today` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/new-schools-today.md`:

**From:**
```
Team of students each with a laptop sitting around the conference table, attentively engege in a video conference call meeting.
```

**To:**
```
New Schools Today
```

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- https://www.hackforla.org/projects/new-schools-today

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/new-schools-today.md`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/new-schools-today.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by kimberlytanyh** [comment]
Availability: Tuesday, Wednesday, Friday (10:00AM-5:00PM)
ETA: Not too sure since I have never done this before. Maybe 5-8 hours to figure out how to do this? Hopefully with lesser time if I pick this up quickly!

**Comment by kimberlytanyh** [comment]
Provide Update
1. Progress: Finished completing action items in issue.
2. Blockers: A few unexpected feedback from terminal when following instructions in Contributing.md. Had to re-do forking and cloning in case so am not sure if committing and pushing was done right.
3. Availability: Same as above.
4. ETA: Around 2-3 hours, excluding time watching YouTube tutorials and reviewing Contributing.md to better understand branching, committing, and pushing on GitHub.
